# motion def:
motion_file: vibrationmotion.csv
# IMU params:
imu: 1
# sample frequency of simulated GNSS/IMU data:
sample_frequency:
    imu: 100.0
    gps: 10.0
# topic name:
imu_topic_name: /sim/sensor/imu
odo_topic_name: /pose/ground_truth
# output rosbag path:
output_path: /workspace/data/gnss_ins_sim/integration
# output name:
output_name: data_vibrationmotion.bag